data/block/block.pb.go

```diff
@@ -186,12 +186,12 @@ func (m *MiniBlock) GetReserved() []byte {
 }
 
 type MiniBlockHeader struct {
-	Hash            []byte `protobuf:"bytes,1,opt,name=Hash,proto3" json:"Hash,omitempty"`
-	SenderShardID   uint32 `protobuf:"varint,2,opt,name=SenderShardID,proto3" json:"SenderShardID,omitempty"`
-	ReceiverShardID uint32 `protobuf:"varint,3,opt,name=ReceiverShardID,proto3" json:"ReceiverShardID,omitempty"`
-	TxCount         uint32 `protobuf:"varint,4,opt,name=TxCount,proto3" json:"TxCount,omitempty"`
-	Type            Type   `protobuf:"varint,5,opt,name=Type,proto3,enum=proto.Type" json:"Type,omitempty"`
-	Reserved        []byte `protobuf:"bytes,6,opt,name=Reserved,proto3" json:"Reserved,omitempty"`
+	Hash            []byte `protobuf:"bytes,1,opt,name=Hash,proto3" json:"hash,omitempty"`
+	SenderShardID   uint32 `protobuf:"varint,2,opt,name=SenderShardID,proto3" json:"senderShardID,omitempty"`
```

Comment:
also here

Reply:
and `ReceiverShardID` ?
also, I would remove also on the `TxCount` field.

Reply:
Removed them